### ISSUE TITLE
Error in NeuralynxRawIO parse_header()

### ISSUE THREAD

**Issue by brykko** [NeuralEnsemble/python-neo#447]
Hi,

I have just been trying to load some Neuralynx tetrode recordings, and I've been reliably getting an error from the `parse_header()` method with various datasets that I've tested. Every time, the error occurs when processing the first .ntt file.

Could it perhaps be down to differences in file format? I'm using Neuralynx Cheetah version 5.6.3. I'm happy to provide some sample data if that helps.
Thanks!
Rich

```python
from neo import rawio

pth = "N:\\richarga\\rec\\2015-11-24_11-58-34"
io = rawio.NeuralynxRawIO(dirname=pth)
io.parse_header()
```
Traceback:

```
N:\richarga\rec\2015-11-24_11-58-34\CSC1.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC10.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC11.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC12.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC13.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC14.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC15.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC2.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC3.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC4.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC5.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC6.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC7.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC8.ncs
N:\richarga\rec\2015-11-24_11-58-34\CSC9.ncs
N:\richarga\rec\2015-11-24_11-58-34\CheetahLogFile.txt
N:\richarga\rec\2015-11-24_11-58-34\CheetahLostADRecords.txt
N:\richarga\rec\2015-11-24_11-58-34\Events.nev
N:\richarga\rec\2015-11-24_11-58-34\TT1.ntt

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-4f2dd9ade201> in <module>()
      7 io = rawio.NeuralynxRawIO(dirname=pth)
----> 8 io.parse_header()

~\AppData\Local\Continuum\Miniconda3\envs\datajoint\lib\site-packages\neo\rawio\baserawio.py in parse_header(self)
    153 
    154         """
--> 155         self._parse_header()
    156         self._group_signal_channel_characteristics()
    157 

~\AppData\Local\Continuum\Miniconda3\envs\datajoint\lib\site-packages\neo\rawio\neuralynxrawio.py in _parse_header(self)
    123                 print("dtype = ", end="")
    124                 print(dtype)
--> 125                 data = np.memmap(filename, dtype=dtype, mode='r', offset=HEADER_SIZE)
    126                 self._spike_memmap[chan_id] = data
    127 

~\AppData\Local\Continuum\Miniconda3\envs\datajoint\lib\site-packages\numpy\core\memmap.py in __new__(subtype, filename, dtype, mode, offset, shape, order)
    234             if (bytes % _dbytes):
    235                 fid.close()
--> 236                 raise ValueError("Size of available data is not a "
    237                         "multiple of the data-type size.")
    238             size = bytes // _dbytes

ValueError: Size of available data is not a multiple of the data-type size.
```

**Comment by JuliaSprenger** [comment]
Hi Richard,
the NeuralynxIO has not been tested on .ntt files since there were none available when I implemented the IO. Currently the code for ntt files is only based on the (sparse) file format documentation provided by Neuralynx. I will have a look at it if you could provide some sample data via email / file sharing.

**Comment by brykko** [comment]
Hi Julia, thanks for offering to take a look. I've emailed you a dropbox link to a sample recording.

**Comment by samuelgarcia** [comment]
I tried to integrate NTT when porting neuralynx to the new NeuralynxRawIO without any ntt file. So it is like shooting in the dark.

@JuliaSprenger: from what I have read in the doc ntt are similar to nse but with tetrode. In the new code, I wrote a generic ```get_nse_or_ntt_dtype``` function. The problem is certainly there.

@brykko: would be OK for putting this dataset (if small enough) on our testing file repo ? Its is publicy availble.

Best


Samuel
